app/routes.js

```diff
@@ -1,7 +1,9 @@
-const { createUser } = require('./controllers/user');
+const { createUser, signIn } = require('./controllers/user');
 const { healthCheck } = require('./controllers/healthCheck');
+const { dataUser, schemaUser } = require('./middlewares/user');
 
 exports.init = app => {
   app.get('/health', healthCheck);
-  app.post('/users', createUser);
+  app.post('/users', schemaUser, [dataUser], createUser);
+  app.post('/users/sessions', signIn);
```

Comment:
Here it would be good to add a middleware that validates that the email and password come but without giving clues about the domain of the email or the length of the password.